doc/Requirements Specification/Parts/Discussion.lyx

```diff
@@ -399,22 +373,9 @@ gls{component}
 
 \end_inset
 
-.
- Whether imitation of other 
-\begin_inset ERT
-status open
-
-\begin_layout Plain Layout
-
-
-\backslash
-glsuserii{component}
-\end_layout
-
-\end_inset
-
- behaviour can be done automatically needs investigation.
- Such imitations may not affect the measurement results.
+ might be very effortful or even impossible.
+ If this problem cannot be solved, the user has to ensure, that all of these
+ are mocket, which need to be.
```

Comment:
I don’t quite like that wording. How about:

“Beagle may require the user to provide a test bed in which the measured components can be executed.”